src/flex.test.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+
+import Flex, {
+	DIRECTION_ROW,
+	DIRECTION_COLUMN,
+	FLEX_CLASS,
+	FLEX_ROW_CLASS,
+	FLEX_COLUMN_CLASS,
+	FLEX_WRAP_CLASS,
+	FLEX_NOGUTTER_CLASS,
+	FLEX_ALIGN_CLASS,
+	VALID_ALIGNMENTS,
+	VALID_BREAKPOINTS,
+	VALID_SPACE,
+} from './Flex';
+
+describe('Flex', function() {
+
+	it('exists', function() {
+		const flex = TestUtils.renderIntoDocument(<Flex />);
+		const flexNode = ReactDOM.findDOMNode(flex);
+
+		expect(flexNode).not.toBeNull();
+	});
+
```

Comment:
nitpick request: keep the spacing in this file consistent. Some `describe`s and `it`s have newlines after them and some don't. Keeping it consistent will make it look cleaner and will soothe my OCD.